### ISSUE TITLE
Issue with spectralsequences and expl3

### ISSUE THREAD

**Issue by sudo-rushil** [hoodmane/spectralsequences#17]
I was trying to use `spectralsequences` on my Mac (literally just copied the intro example into my pset document), and I got this error:

```
/usr/local/texlive/2021/texmf-dist/tex/latex/spectralsequences/sseqcheckdefinitions.code.tex:78: Package spectralsequences Error: Internal error: Since this version of
(spectralsequences)                spectralsequences, the expl3 definition
(spectralsequences)                of \peek_meaning_ignore_spaces:NTF  was
(spectralsequences)                updated in an incompatible way. Contact
(spectralsequences)                package author Hood Chatham at
(spectralsequences)                hood@mit.edu.
```

I tried updating all my TexLive packages (`tlmgr update --all`) but that didn't help; not sure how to proceed from here...

**Comment by hoodmane** [comment]
Thanks for the report! What do you see if you run 
`cat /usr/local/texlive/2021/texmf-dist/tex/latex/spectralsequences/spectralsequences.sty  | head`?

**Comment by sudo-rushil** [comment]
I get
```
%%
%% Package: spectralsequences v1.3.1 2022-01-04
%% Author: Hood Chatham
%% Email: hood@mit.edu
%% Date: 2022-01-04
%% License: Latex Project Public License
%%
%% A package for drawing spectral sequences
%%
```

**Comment by hoodmane** [comment]
Okay, that's up to date.
What about the output from:
```
echo x | pdflatex "\RequirePackage{expl3}\ExplSyntaxOn\show\peek_meaning_ignore_spaces:NTF"
```

**Comment by sudo-rushil** [comment]
```
This is pdfTeX, Version 3.141592653-2.6-1.40.23 (TeX Live 2022/dev) (preloaded format=pdflatex 2022.2.18)  18 FEB 2022 15:27
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**\RequirePackage{expl3}\ExplSyntaxOn\show\peek_meaning_ignore_spaces:NTF

(/usr/local/texlive/2021/texmf-dist/tex/latex/l3kernel/expl3.sty
Package: expl3 2022-02-05 L3 programming layer (loader)

(/usr/local/texlive/2021/texmf-dist/tex/latex/l3backend/l3backend-pdftex.def
File: l3backend-pdftex.def 2022-02-07 L3 backend support: PDF output (pdfTeX)
\l__color_backend_stack_int=\count185
\l__pdf_internal_box=\box50
))
> \peek_meaning_ignore_spaces:NTF=\protected\long macro:
->\__kernel_if_debug:TF {\msg_warning:nnxxx {deprecation}{deprecated-command}{2
022-01-11}{\peek_meaning_ignore_spaces:NTF}{\peek_remove_spaces:n }}{}\cs_gset_
protected:Npn \peek_meaning_ignore_spaces:NTF ##1##2##3{\peek_remove_spaces:n {
\peek_meaning:NTF ##1{##2}{##3}}}\peek_meaning_ignore_spaces:NTF .
<*> ...yntaxOn\show\peek_meaning_ignore_spaces:NTF
                                                  ^^M
? x
```

**Comment by hoodmane** [comment]
Okay, well they added a deprecation message to it.

**Comment by hoodmane** [comment]
For now you could patch `/usr/local/texlive/2021/texmf-dist/tex/latex/spectralsequences/sseqcheckdefinitions.code.tex` by deleting lines 71 -- 76. Or you could downgrade to latex3 version [2021-12-14](https://github.com/latex3/latex3/releases/tag/2021-12-14) before they deprecated it but that's probably the wrong approach.

I will probably make a bugfix release with this in the next few days.

I really need to remove these stupid expl3 patches, they are definitely a problem see #12.

**Comment by sudo-rushil** [comment]
Yep, commenting out those lines fixed everything! Thank you so much!!

Let me know if I should close the issue, or if you want it to stay open until it gets patched

**Comment by hoodmane** [comment]
Definitely don't close the issue =)

**Comment by hoodmane** [comment]
Well I added the `texlive-latest` to the CI, and I am getting this failure in CI now =)
https://app.circleci.com/pipelines/github/SpectralSequences/latex/100/workflows/bfbe76d6-a111-4b6b-af6f-7377f4cc7efb/jobs/1550

**Comment by hoodmane** [comment]
I released v1.3.2 with a fix for this. Should be available in the next few days.
